doc/help/MERGE.md

```diff
@@ -0,0 +1,44 @@
+kdb-merge(1) -- Three-way merge of KeySets.
+================================
+## DESCRIPTION
+Does a three-way merge between keysets.  
+On success the resulting keyset will be saved to mergepath.  
+On unresolved conflicts nothing will be changed.  
+
+## USAGE
```

Comment:
Thats SYNOPSIS and typically before DESCRIPTION. Its best if you look at other man pages how they do it.
This also confused me in the beginning (its starts with three-way merge without giving SYNOPSIS)

Text is nice, though!